main.go

```diff
@@ -178,6 +181,8 @@ func main() {
 		}
 		if !exist && err == nil {
 			klog.Infof("cert-manager CRD does not exist, skip cert-manager related controllers initialization")
+			// start go routines
+			ch <- bs
```

Comment:
Hi @qpdpQ 
I am wondering that are we able to send the single right after we setup manager for `CommonServiceReconciler{}` at line 175?
```
                if err = (&controllers.CommonServiceReconciler{
			Bootstrap: bs,
			Scheme:    mgr.GetScheme(),
			Recorder:  mgr.GetEventRecorderFor("commonservice-controller"),
		}).SetupWithManager(mgr); err != nil {
			klog.Errorf("Unable to create controller CommonService: %v", err)
			os.Exit(1)
		}
                
                ch <- bs
```